server/views/applications/show.njk

```diff
@@ -60,17 +68,50 @@
           text: 'Placement requests',
           href: ApplyUtils.applicationShowPageTab(application.id, ApplyUtils.applicationShowPageTabs.placementRequests),
           active: tab === ApplyUtils.applicationShowPageTabs.placementRequests
-        }]
+        }
+        ]
       }) }}
+      </div>
 
-      {% if tab === ApplyUtils.applicationShowPageTabs.timeline %}
-        {{ timeline(mapTimelineEventsForUi(timelineEvents), application, csrfToken) }}
-      {% elif tab === ApplyUtils.applicationShowPageTabs.placementRequests %}
-        {{ requestAPlacement(placementApplications, application, user, csrfToken) }}
-      {% else %}
-        {{ applicationReadonlyView(application) }}
-      {% endif %}
+      {% if tab === ApplyUtils.applicationShowPageTabs.application %}
+        <div class="govuk-grid-column-one-quarter govuk-!-padding-left-0 govuk-!-padding-right-0">
+
+          {{ mojIdentityBar({
```

Comment:
If you look at the bookings/show.njk template, the actions are to the right of the heading e.g:

![image](https://github.com/ministryofjustice/hmpps-approved-premises-ui/assets/109774/703d1f1d-dd97-4235-ab1f-c288e82ca15b)

I think we should be consistent with that approach. Happy to be wrong though!

Reply:
OK I get you. 
It's not what the mocks look like but it would be much simpler for us so I think we should go with it for now

Reply:
Cool - yeah, I think it makes sense to lean on existing patterns that are in use elsewhere 👍